From a82d704b1ec6ece47b01d12e0e067d4b62b10894 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Wed, 9 Nov 2022 20:24:45 +0100
Subject: [PATCH] m4/ax_python.m4: check for python-x.y-emded.pc, not
 python-x.y.pc

Only the embed version includes necessary linker flags to link
with libpython.

Upstream-Status: Backport
[https://github.com/FRRouting/frr/commit/a82d704b1ec6ece47b01d12e0e067d4b62b10894]

Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 m4/ax_python.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/ax_python.m4 b/m4/ax_python.m4
index 91d12b99b..f5e603b96 100644
--- a/m4/ax_python.m4
+++ b/m4/ax_python.m4
@@ -206,7 +206,7 @@ AC_REQUIRE([PKG_PROG_PKG_CONFIG])dnl
     AC_MSG_CHECKING([whether pkg-config python-${tryver} is available])
     unset PYTHON_CFLAGS
     unset PYTHON_LIBS
-    pkg="python-${tryver}"
+    pkg="python-${tryver}-embed"
     pkg="${pkg%-}"
     _PKG_CONFIG([PYTHON_CFLAGS], [cflags], [${pkg}])
     _PKG_CONFIG([PYTHON_LIBS], [libs], [${pkg}])
-- 
2.25.1

